sdk/translation/azure-ai-translation-document/tests/testcase.py

```diff
@@ -130,12 +130,15 @@ def wait(self, duration=30):
 
 
     # model helpers
-    def _validate_doc_status(self, doc_details, target_language):
+    def _validate_doc_status(self, doc_details, target_language, **kwargs):
+        status = kwargs.pop("statuses", ["Succeeded"])
+        ids = kwargs.pop("statuses", None)
         # specific assertions
-        self.assertEqual(doc_details.status, "Succeeded")
+        self.assertIn(doc_details.status, status)
```

Comment:
This might break other tests where we were just passing the string status. You might have to check if the status is an instance of a list before calling self.assertIn

Reply:
actually no!
i made sure that every instance call was passing parameters correctly
also before this, i used to assert for 'succeeded' without passing the doc status